src/components/ContextBar.test.js

```diff
@@ -331,12 +331,12 @@ const contextBarProps = {
 describe('ContextBar - buttons', () => {
   it('will render default icons', () => {
     // Act
-    const { getByText, getByLabelText } = render(h(ContextBar, contextBarProps))
+    const { getByText, getByLabelText, getByTestId } = render(h(ContextBar, contextBarProps))
```

Comment:
Having to access the button by test ID is not ideal, but there is no visible text to go off of. Previously an aria-label was being added to an element that does not support that field (an anchor tag),[ resulting in an a11y error](https://broadworkbench.atlassian.net/browse/IA-3369). I have added screenreader-only text so that the link reads out nicely, but screenreader-only text is not used by getText nor getByLabelText.

FYI @n8landolt